src/neuroconv/basedatainterface.py

```diff
@@ -108,6 +159,38 @@ def run_conversion(
             nwbfile=nwbfile,
             metadata=metadata,
             overwrite=overwrite,
+            backend=backend,
             verbose=getattr(self, "verbose", False),
         ) as nwbfile_out:
-            self.add_to_nwbfile(nwbfile_out, metadata=metadata, **conversion_options)
+            if backend_configuration is None:  # Otherwise Assume the data has already been added to the NWBFile
+                # Otherwise assume the data has already been added to the NWBFile
+                self.add_to_nwbfile(nwbfile_out, metadata=metadata, **conversion_options)
+
+                backend_configuration = self.get_default_backend_configuration(
+                    nwbfile=nwbfile_out, backend=backend, **conversion_options
+                )
+
+            configure_backend(nwbfile=nwbfile_out, backend_configuration=backend_configuration)
+
+    @staticmethod
+    def get_default_backend_configuration(
+        nwbfile: NWBFile,
+        backend: Literal["hdf5", "zarr"] = "hdf5",
+        **conversion_options,
```

Comment:
why have conversion options here?

Reply:
Whoops - good catch. Thought I had removed all of that

Reply:
Caught a couple other missed propagations, think that should be all of them